### PR TITLE
add query parameters for `expand`

### PR DESCRIPTION
**Pull request by albertchen-okta**
## Description:
- **What's changed?** 

new query parameter for GET and LIST email templates 

- **Is this PR related to a Monolith release?** 
- 2022.08.2

### Resolves:

* https://oktainc.atlassian.net/browse/OKTA-497965

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/brands/index.md [hunk 1]
```diff
@@ -1303,6 +1305,12 @@ HTTP/1.1 200 OK
 {
     "name": "UserActivation",
     "_links": {
+    "_embedded": {
+      "settings": {
+        "recipients": "ALL_USERS",
+        "_links": { ... }
+      },
+      "customizationCount": 3
```
Comment (gormanho-okta) [inline packages/@okta/vuepress-site/docs/reference/api/brands/index.md:1313]: The `_embedded` block should be a sibling to the `_links` blocks (i.e., at the same level):
```
{
  "name": ...
  "_links": { ... },
  "_embedded": {
    "settings": ...
    "customizationCount": ...
  }
}
```

File: packages/@okta/vuepress-site/docs/reference/api/brands/index.md [hunk 2]
```diff
@@ -1181,9 +1181,10 @@ Lists all supported email templates
 
 #### Request path parameters
 
-| Parameter      | Type        | Description            |
-| -------------- | ----------- | ---------------------- |
-| `brandId`      | String      | ID of a Brand          |
+| Parameter      | Description            | ParamType | DataType | Required |
+| -------------- | ---------------------- | --------- | -------- | -------- |
+| `brandId`      | ID of a Brand String   | Query     | String   | TRUE     |
+| `expand`       | If specified, it causes additional metadata to be included in the response. Possible values are `settings` and/or `customizationCount`.           | Query     | String    | FALSE     |
```
Comment (gormanho-okta) [inline packages/@okta/vuepress-site/docs/reference/api/brands/index.md:1187]: I know the example already illustrates it, but should we mention here that if both are specified, that they should be separated by a comma?

Comment (albertchen-okta) [reply packages/@okta/vuepress-site/docs/reference/api/brands/index.md:1187]: done

**Review by gormanho-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/reference/api/brands/index.md [hunk 3]
```diff
@@ -1181,9 +1181,10 @@ Lists all supported email templates
 
 #### Request path parameters
 
-| Parameter      | Type        | Description            |
-| -------------- | ----------- | ---------------------- |
-| `brandId`      | String      | ID of a Brand          |
+| Parameter      | Description            | ParamType | DataType | Required |
+| -------------- | ---------------------- | --------- | -------- | -------- |
+| `brandId`      | ID of a Brand String   | Query     | String   | TRUE     |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/brands/index.md:1186]:
```suggestion
| `brandId`      | ID of a Brand string   | Query     | String   | TRUE     |
```

File: packages/@okta/vuepress-site/docs/reference/api/brands/index.md [hunk 4]
```diff
@@ -1181,9 +1181,10 @@ Lists all supported email templates
...
+| `expand`       | If specified, it causes additional metadata to be included in the response. Possible values are `settings` and/or `customizationCount`. Use commas to separate values if both are used.          | Query     | String    | FALSE     |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/brands/index.md:1187]:
```suggestion
| `expand`       | If specified, it causes additional metadata to be included in the response. Supported values: `settings` and/or `customizationCount`. Use commas to separate values if both are used.          | Query     | String    | FALSE     |
```

File: packages/@okta/vuepress-site/docs/reference/api/brands/index.md [hunk 5]
```diff
@@ -1268,10 +1269,11 @@ Fetches the email template named `templateName`
 
 #### Request path parameters
 
-| Parameter      | Type        | Description               |
-| -------------- | ----------- | ------------------------- |
-| `brandId`      | String      | ID of a Brand             |
-| `templateName` | String      | Name of an Email Template |
+| Parameter      | Description            | ParamType | DataType | Required |
+| -------------- | ---------------------- | --------- | -------- | -------- |
+| `brandId`      | ID of a Brand String   | Query     | String   | TRUE     |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/brands/index.md:1274]:
```suggestion
| `brandId`      | ID of a Brand string   | Query     | String   | TRUE     |
```

File: packages/@okta/vuepress-site/docs/reference/api/brands/index.md [hunk 6]
```diff
@@ -1268,10 +1269,11 @@ Fetches the email template named `templateName`
...
+| `templateName` | Name of an Email Template | Query     | String   | TRUE     |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/brands/index.md:1275]:
```suggestion
| `templateName` | Name of an email template | Query     | String   | TRUE     |
```

File: packages/@okta/vuepress-site/docs/reference/api/brands/index.md [hunk 7]
```diff
@@ -1268,10 +1269,11 @@ Fetches the email template named `templateName`
...
+| `expand`       | If specified, it causes additional metadata to be included in the response. Possible values are `settings` and/or `customizationCount`. Use commas to separate values if both are used.          | Query     | String    | FALSE     |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/brands/index.md:1276]:
```suggestion
| `expand`       | If specified, it causes additional metadata to be included in the response. Supported values: `settings` and/or `customizationCount`. Use commas to separate values if both are used.          | Query     | String    | FALSE     |
```

**Review by susanharper-okta — APPROVED** [review]
couple nits - approving to not hold you up.

**Pull request merged**